### PR TITLE
#1051 add ctor with vargs String

### PR DESCRIPTION
**Pull request by kokodyn**
Change related with #1051. I see i has been partly covered in PR #1480. Adding ctor which accepts String vargs in ctor is also required.

### CODE REVIEW

File: src/main/java/org/cactoos/text/Concatenated.java [hunk 1]
```diff
@@ -43,6 +44,19 @@ public Concatenated(final Text... txts) {
...
 
+    /**
+     * Ctor.
+     * @param strs Strings to be concatenated
+     */
+    public Concatenated(final String... strs) {
+        this(
+            new Mapped<>(
+                TextOf::new,
+                new IterableOf<String>(strs)
```
Comment (andreoss) [inline src/main/java/org/cactoos/text/Concatenated.java:55]: @kokodyn Do you need explicit type parameter here?

**Review by andreoss — CHANGES_REQUESTED** [review]

File: src/main/java/org/cactoos/text/Concatenated.java (hunk 1 above)
Comment (kokodyn) [reply src/main/java/org/cactoos/text/Concatenated.java]: @andreoss fixed

**Comment by victornoel** [comment]
@rultor merge

**Comment by victornoel** [comment]
@kokodyn @andreoss thanks

**Pull request merged**